### PR TITLE
Info for The input device is not a TTY Errors

### PR DESCRIPTION
**Pull request by holzhannes**
Give a hint to use "-t" flag for exec to use them in cron jobs

### CODE REVIEW

**Review by holzhannes — COMMENTED** [review]
Changed position of information for better understanding

File: docs/administration.rst [hunk 1]
```diff
@@ -240,7 +238,9 @@ Document exporter
 =================
 
 The document exporter exports all your data from paperless into a folder for
-backup or migration to another DMS.
+backup or migration to another DMS. 
+
+If you use the document exporter within a cronjob to backup your data you might use the ``-t`` flag behind exec to suppress "The input device is not a TTY" errors. For example: ``docker-compose exec -t webserver document_exporter ../export``
```
Comment (sbrunner) [inline docs/administration.rst:243]:
```suggestion
If you use the document exporter within a cronjob to backup your data you might use the ``-T`` flag behind exec to suppress "The input device is not a TTY" errors. For example: ``docker-compose exec -T webserver document_exporter ../export``
```

Comment (holzhannes) [reply docs/administration.rst:243]: Thank you for correcting the typo!

Comment (jonaswinkler) [reply docs/administration.rst:243]: That's quite the typo! :)

Comment (jonaswinkler) [reply docs/administration.rst:243]: Doesn't this apply to all management commands that have TTY output?

**Comment by jonaswinkler** [comment]
Doesn't this apply to all management commands that have TTY output, not just the document exporter?

**Comment by sbrunner** [comment]
Yes docker-compose will fail it we don't provide the -T option, and he can't "communicate" with an interactive terminal...

**Comment by jonaswinkler** [comment]
In that case, I'd propose to make the change further above instead. There's a section that describes how to run any management command with docker compose.

**Pull request merged**